extension/src/openvic-extension/singletons/MenuSingleton.hpp

```diff
@@ -117,11 +123,12 @@ namespace OpenVic {
 		godot::Error population_menu_select_province(int32_t province_index);
 		godot::Error population_menu_toggle_expanded(int32_t toggle_index, bool emit_selected_changed = true);
 
-		void _population_menu_update_pops();
-		void _population_menu_update_filtered_pops();
+		bool _population_menu_update_pops();
```

Comment:
I don't understand what the bool indicates. Neither the function name, context or comments explain it.

Reply:
It's the second case, true = no failure, false = failure. I used `bool` for functions which can't be called from GDScript, following the same pattern as we use in the SIM repo, but since these functions are all called by GDScript-accessible functions I might as well use `godot::Error` right away.